lib/ansible/modules/cloud/amazon/rds_snapshot.py

```diff
@@ -2,29 +2,35 @@
 # Copyright (c) 2014 Ansible Project
 # Copyright (c) 2017, 2018, 2019 Will Thames
 # Copyright (c) 2017, 2018 Michael De La Rue
+# Copyright (c) 2019 Joel Stuedle
 # GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
 
 from __future__ import absolute_import, division, print_function
 __metaclass__ = type
 
-
 ANSIBLE_METADATA = {'status': ['preview'],
                     'supported_by': 'community',
                     'metadata_version': '1.1'}
-
 DOCUMENTATION = '''
 ---
 module: rds_snapshot
-version_added: "2.9"
-short_description: manage Amazon RDS snapshots.
+version_added: '2.9'
+short_description: manage Amazon RDS/Aurora snapshots.
 description:
      - Creates or deletes RDS snapshots.
 options:
+  snapshot_type:
```

Comment:
Can we not just assume that passing `db_cluster_identifier` means `snapshot_type` is aurora?